deploy/charts/external-secrets/values.yaml

```diff
@@ -338,3 +338,9 @@ certController:
       # requests:
       #   cpu: 10m
       #   memory: 32Mi
+
+
+dnsOptions:
+  options:
+    - name: ndots
+      value: "1"
```

Comment:
We can not change the default setting from `ndots:5` to `ndots:1`. 
Also i'd prefer having the option in `values.yaml` empty, so it doesn't add any dnsOptions by default.

Reply:
Hello @moolen thank you for response.

 I removed default values from values.yaml now and README.md adjusted as well.